packages/compass-home/src/components/home.tsx

```diff
@@ -392,6 +393,8 @@ function ThemedHome(
   const [isWelcomeOpen, setIsWelcomeOpen] = useState(false);
 
   useLayoutEffect(() => {
+    resetGlobalCSS();
```

Comment:
This doesn't need to be called from react lifecycle, I think we probably just want to immediately call in in compass renderer before the application rendered anything on the screen. If you don't want to add components as a dependency to compass package, you can just call it right after import in the module scope

Reply:
Moved to the top of this file. Open to moving it elsewhere if you think there's a better spot. I'm thinking it's nice to keep compass-components out of the compass package.